README.md

```diff
@@ -1,7 +1,5 @@
 # Software for Open Networking in the Cloud (SONiC)
 
-This is the repository for the Web site for the Software for Open Networking in the Cloud (SONiC)
```

Comment:
Hi Amy, this repository really is just for the sonic website.  :)  There are many other repositories which contain the source code for the various pieces of the project.   I'd like to leave this file as is.